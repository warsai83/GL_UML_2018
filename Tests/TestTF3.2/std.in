LOAD e2.txt
ANALYSE e.txt
QUITTER
