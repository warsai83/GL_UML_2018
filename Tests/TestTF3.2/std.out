Warning, aucune maladie ne correspond Ã  l'empreinte
Maladies connues : 
Maladie1
Maladie2
Maladie3